lib/koans/10_map_sets.ex

```diff
@@ -15,8 +15,11 @@ defmodule MapSets do
     assert MapSet.size(new_set) == ___
   end
 
-  koan "Even though I am like a list I am unordered after 32 elements" do
-    assert 1..33 |> MapSet.new() |> Enum.fetch(0) == {___, 11}
+  koan "Even though I am like a list I am unordered, especially with 33+ elements" do
```

Comment:
As you mention in #169, the number "32" is unexplained. TBH I don't remember what's special about that either. I wonder if a comment close by with more explanation might be interesting to have?